examples/mnist/train.cpp

```diff
@@ -54,53 +59,52 @@ static void construct_net(network<sequential>& nn) {
     // C : convolution
     // S : sub-sampling
     // F : fully connected
-    nn << convolutional_layer<tan_h>(32, 32, 5, 1, 6,  // C1, 1@32x32-in, 6@28x28-out
-            padding::valid, true, 1, 1, backend_type)
-       << average_pooling_layer<tan_h>(28, 28, 6, 2)   // S2, 6@28x28-in, 6@14x14-out
-       << convolutional_layer<tan_h>(14, 14, 5, 6, 16, // C3, 6@14x14-in, 16@10x10-out
-            connection_table(tbl, 6, 16),
-            padding::valid, true, 1, 1, backend_type)
-       << average_pooling_layer<tan_h>(10, 10, 16, 2)  // S4, 16@10x10-in, 16@5x5-out
-       << convolutional_layer<tan_h>(5, 5, 5, 16, 120, // C5, 16@5x5-in, 120@1x1-out
-            padding::valid, true, 1, 1, backend_type)
-       << fully_connected_layer<tan_h>(120, 10,        // F6, 120-in, 10-out
-            true, backend_type)
-    ;
+    nn << convolutional_layer<tan_h>(32, 32, 5, 1,
+                                     6,  // C1, 1@32x32-in, 6@28x28-out
+                                     padding::valid, true, 1, 1, backend_type)
+       << average_pooling_layer<tan_h>(28, 28, 6,
+                                       2)  // S2, 6@28x28-in, 6@14x14-out
+       << convolutional_layer<tan_h>(14, 14, 5, 6,
+                                     16,  // C3, 6@14x14-in, 16@10x10-out
+                                     connection_table(tbl, 6, 16), padding::valid, true, 1, 1, backend_type)
+       << average_pooling_layer<tan_h>(10, 10, 16,
+                                       2)  // S4, 16@10x10-in, 16@5x5-out
+       << convolutional_layer<tan_h>(5, 5, 5, 16,
+                                     120,  // C5, 16@5x5-in, 120@1x1-out
+                                     padding::valid, true, 1, 1, backend_type)
+       << fully_connected_layer<tan_h>(120, 10,  // F6, 120-in, 10-out
+                                       true, backend_type);
```

Comment:
It definitely got worse, but I even don't want to put `//clang-format off` since unless we think about different formatting which is not totally horrible